proxy/src/telemetry/metrics/mod.rs

```diff
@@ -109,8 +109,14 @@ struct Root {
 }
 
 #[derive(Debug)]
-struct Scopes<L: Display + Hash + Eq, M> {
-    scopes: IndexMap<L, M>,
+struct Scopes<L: Display + Hash + Eq, S> {
+    scopes: IndexMap<L, S>,
+}
+
+#[derive(Debug)]
```

Comment:
Should this derive `Copy`/`Clone`?

Reply:
There's no use case for duplicating metrics registries, so I think it's better to omit until we have a need.